Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.h

```diff
@@ -75,6 +75,10 @@ namespace O3DE::ProjectManager
         inline constexpr static int s_platformTextLineBottomMargin = 5;
         inline constexpr static int s_platformTextWrapAroundLineMaxCount = 2;
 
+        // Version
+        inline constexpr static int s_versionSize = 50;
+        inline constexpr static int s_versionSizeSpacing = 25;
```

Comment:
The `static` keyword isn't needed.
internal linkage isn't needed for each constant.

Reply:
looks like inline cannot be used without 'static' and these vars are referred to in other .cpp files like `GemCatalogScreen.cpp` and `GemListView.cpp` .  I could make accessor functions, but I might leave that for another PR.

Reply:
Oh, these are static class members.
Never mind then.
I thought these were non-member constexpr constants.